src/Propel/Runtime/Connection/StatementWrapper.php

```diff
@@ -220,24 +220,15 @@ public function execute($inputParameters = null)
      *
      * @return string
      */
-    public function getExecutedQueryString($inputParameters = null)
+    public function getExecutedQueryString(?array $inputParameters = null): string
```

Comment:
If this is planned to be released as part of the upcoming beta release, we could think about keeping those, and also add other missing ones.
For a pure bugfix release it would be a bit too much.

cc @mringler

Reply:
That's a good point. That mean there is no better time to add them than now, doesn't it?

Tell me if I am missing something: the big problem with adding type hints is how easily they cause errors in seemingly unrelated parts of the code, making it hard to see the true impact of such a change. With stan, psalm and the tests, we should be relatively safe, but those are famous last words. I would guess that every new type hint is still a crapshoot to some extend. So if we add enough, at some point, we will inevitably end up with a bug report.
Then again, the type hints won't add themselves. And while they can make things less flexible, it is probably still better to have them in almost all cases.

Personally, I think I like seeing them added. But I must admit, it is dicey and I am not sure if I am even completely aware of the extend of trouble it can create. I tend to say let's find out, and accept the risk of having to add a few quick fixes sometimes in the near future. It is the best time for that. But I'd completely understand if you decide that the risk outweighs the benefit.

Reply:
It should not break anything or at least if it breaks smth, that means that smth was broken from the beginning.

But mb we can also add this method to the interface if we started to refactor it?